draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -9280,6 +9280,42 @@ Content-Encoding: gzip
 </t>
 </section>
 
+<section title="Disclosure of Information through DNS Requests" anchor="dns.disclosure">
+<t>
+   Many HTTP applications use the Domain Name System (DNS) {{?RFC1035}} to
+   resolve the authority for HTTP and HTTPS URIs.
+</t>
+<t>
+   Unencrypted DNS requests necessarily expose the activity of the user to
+   network observers. Because Web browsing often requires resolving many
+   hosts to load a page -- the pattern of which often being unique to that
+   page, and possibly even unique to the user -- this can result in the
+   disclosure of a substantial amount of information. Possible mitigations
+   include supporting encrypted DNS protocols, such as {{?RFC8484}}. See also
+   {{?RFC7626}}.
+<t>
+   Furthermore, such resolution needs to adhere to any special requirements
+   documented in the Special-Use Domain Names registry {{!RFC6761}} which
+   might specify that some protocol other than DNS be used for resolution for
+   names within a particular domain. If those special requirements are not
+   honoured diligently, security, privacy and interoperability problems might
```

Comment:
Writing "then security" would improve clarity a bit.

Also, I'd recommend using an Oxford comma, but maybe that's consistently not done for HTTP?